packages/tsdocs/.gitignore

```diff
@@ -0,0 +1,2 @@
+/fixtures/monorepo/docs
```

Comment:
I believe we are not using package-local `.gitignore` files and specify all ignore rules in the monorepo-level `.gitignore` file. Please fix.